_posts/2016-07-11-value-of-automated-e2e-testing.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+author: riikkanen
+title: >
+  The Value of Automated e2e-testing in a Software Project
+excerpt: >
+  A peek into automated e2e-testing done in a software project. The first try with Wicket Page Tester. The second and more sophisticated version with JBehave and Selenium.
+tags:
+- Java
+- testing
+- e2e testing
+- JBehave
+- Wicket
+- Selenium
+- programming
+---
+
+The software project concerned started in the beginning of Autumn in 2010. I had the honor to work in the project as a full stack developer from its beginning to June 2014 when my maternity leave began. The project was about to renew a 30 years old, technically outdated registry and everything attached to it, such as the user inteface and all the integrations. The project was a success, after almost three years of hard work, the application was taken into serious use without any major hardships. The project is still alive and kicking, even though, nowadays mostly maintenance, but also some feature development. 
```

Comment:
Autumn in lowercase, drop the in between autumn and 2010.

"The project was about to renew" -> The project's goal was?

"The project was a success, after almost three years of hard work, the application was taken into serious use without any major hardships." -> After three years of hard work the application was successfully taken into use in production?

"The project is still alive and kicking, even though, nowadays mostly maintenance, but also some feature development. " -> Lots of commas: "The project is still alive and kicking in maintenance mode with some small feature development"?